packages/grid/_modules_/grid/components/containers/GridRoot.tsx

```diff
@@ -27,7 +27,7 @@ export const GridRoot = React.forwardRef<HTMLDivElement, GridRootProps>(function
       role="grid"
       aria-colcount={visibleColumnsLength}
       aria-rowcount={gridState.rows.totalRowCount}
-      tabIndex={0}
+      tabIndex={-1}
```

Comment:
Should we do this here to tackle point 2 from the ticket?

Reply:
The correct state should be:

```suggestion
```